packages/jsii-pacmak/lib/targets/python.ts

```diff
@@ -655,14 +656,15 @@ class TypedDict extends BasePythonClassType {
 
             // We'll emit the optional members first, just because it's a little nicer
             // for the final class in the chain to have the mandatory members.
+            code.line(`@jsii.data_type_optionals(jsii_struct_bases=[${baseInterfaces.join(', ')}])`);
             code.openBlock(`class _${this.name}(${classParams.concat(["total=False"]).join(", ")})`);
             for (const member of optionalMembers) {
                 member.emit(code, resolver);
             }
             code.closeBlock();
 
             // Now we'll emit the mandatory members.
-            code.line(`@jsii.data_type(jsii_type="${this.fqn}")`);
+            code.line(`@jsii.data_type(jsii_type="${this.fqn}", jsii_struct_bases=[_${this.name}])`);
```

Comment:
What's the `_` for here?

Reply:
Every struct gets compiled into 2 classes:

```python
class _MyStruct(...):
  pass

class MyStruct(_MyStruct):
  pass
```

One holds the required fields, the other one the optional fields.